Kajabi/sage-lib
# SageLink updates

## Description
This PR updates the Sage Link rails implementation:
1. Uses `link_to` under-the-hood to generate links
2. Now has a `component.content` slot for adding custom/nested content
3. General refactoring and cleanup

## Screenshots
|  Before  |  After  |
|--------|--------|
|<img width="1792" alt="Before" src="https://user-images.githubusercontent.com/7331038/130256867-6cc8544e-5811-4ddb-adbe-11ad422dfe63.png">|<img width="1792" alt="After" src="https://user-images.githubusercontent.com/7331038/130281148-400cf710-7ff0-450d-8785-499988b4da36.png">|

## Testing in `sage-lib`
1. Visit http://0.0.0.0:4000/pages/component/link and ensure that the screen matches the `After` screenshot
2. Ensure all links are rendering properly in the DOM

## Testing in `kajabi-products`
1. (**HIGH**) Any use of the `SageLink` Rails component will be affected.
   - [ ] Account Password - /admin/settings/password
   - [ ] Contact Info - /admin/settings/contact_info
   - [ ] Account Users - /admin/settings/users
